frontend/src/components/latest-post-card.tsx

```diff
@@ -24,7 +24,14 @@ export default function LatestPostCard({
             <CategoryPill key={`${category}-${index}`} category={category} />
           ))}
         </div>
-        <img src={linkIcon} alt={post.title} className="h-3 w-3" />
+        {/* <img src={linkIcon} alt={post.title} className="h-3 w-3" /> */}
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
```

Comment:
Instead of embedding this as a svg, create file and import from there, like the existing linkIcon.
Ideally rename the existing linkIcon to link-light-icon and have a new one link-dark-icon

Reply:
I can give containing SVG in assests and use it like the previous embedded linkIcon one more try.

Reply:
We can't use context api, as it renders every component beneath, we can use zustand state management instead, if you can stick to the contributing guidelines and do nice refactoring @Ashu0216

Reply:
Sure, will try to utilize zutsand, and will keep you updated of the progress @krishnaacharyaa